lisa/notifiers/json.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
```

Comment:
Does it include platform information like image and vm size?

Reply:
it contains vmsize, kernel version, it doesn't contain image info.

```
{
    "type": "Performance",
    "time": "2024-04-10 07:25:38.061040",
    "elapsed": 0,
    "tool": "iperf3",
    "test_case_name": "perf_tcp_iperf_sriov",
    "platform": "azure",
    "location": "westus2",
    "host_version": "20348-10.0-1-0.1606",
    "guest_os_type": "Linux",
    "distro_version": "Ubuntu 20.04.6 LTS",
    "vmsize": "Standard_DS2_v2",
    "kernel_version": "5.15.0-1060-azure",
    "lis_version": "",
    "ip_version": "IPv4",
    "protocol_type": "TCP",
    "data_path": "Sriov",
    "test_date": "2024-04-10 07:17:08.413022",
    "role": "",
    "test_result_id": "lisa_0_0",
    "connections_num": 1,
    "throughput_in_gbps": "0",
    "latency_us": "0",
    "buffer_size": "0",
    "tx_packets": "0",
    "rx_packets": "0",
    "pkts_interrupts": "0",
    "number_of_receivers": 1,
    "number_of_senders": 1,
    "sender_cycles_per_byte": "0",
    "connections_created_time": 0,
    "retrans_segments": 0,
    "receiver_cycles_rer_byte": "0",
    "buffer_size_bytes": "32",
    "tx_throughput_in_gbps": 0.09070724171712838,
    "rx_throughput_in_gbps": 0.09070724171712838,
    "retransmitted_segments": 0,
    "congestion_windowsize_kb": "69.4904296875"
}
```